Cargo.toml

```diff
@@ -49,7 +50,7 @@ zeroize = { version = "1.5", features = ["zeroize_derive"], optional = true }
 
 # Unexported dependnecies
 bitcoin_hashes = { version = ">=0.12, <=0.13", default-features = false }
-unicode-normalization = { version = "=0.1.22", optional = true }
+unicode-normalization = { version = "=0.1.22", default-features = false, optional = true }
```

Comment:
Oh I did not notice before but I don't think this should be a fixed version? Should be `version = "0.1.22"`.

Reply:
No then it will upgrade and this crate has repeatedly broken MSRV for some of my projects. It's an internal dep so the version used shouldn't matter. Luckily 0.1.22 now is over a year old and seems like he saturated the project too. Which is ok :)

Reply:
It makes it a bit hard to package on distributions though if dependencies get out of sync, one would have to patch the `Cargo.toml` file to allow other versions.